pages/linux/xev.md

```diff
@@ -0,0 +1,20 @@
+# xev
+
+> Print contents of X events.
+> More information: <https://gitlab.freedesktop.org/xorg/app/xev>.
+
+- View all occuring X events:
+
+`xev`
+
+- Monitor the X events of the root window instead of creating a new one:
+
+`xev --root`
+
+- View the X events of a particular window:
+
+`xev -id {{Window_id}}`
+
+- View a category of X events (multiple flags can be chained):
+
+`xev --event {{Event_category}}`
```

Comment:
```suggestion
- Monitor a category of X events (can be specified multiple times):

`xev --event {{event_category}}`
```
Is there a way to get a list of all event categories. I found those in the man page, but we can't really include all of them: `keyboard mouse expose visibility structure substructure focus property colormap owner_grab_button randr button`

Reply:
I don't know if there's a way to get that list other than from the man page.

Reply:
Then let's just leave them out.